app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "AllAboutOlaf",
+  "displayName": "All About Olaf"
+}
```

Comment:
I thought so too, but that was only introduced eight days ago? It's not in _any_ tags yet.